src/controllers/core/controller.js

```diff
@@ -59,6 +59,8 @@ class Controller
 
                 if (append) {
                     $(anchor).append(template);
+                } else if (prepend) {
```

Comment:
I had hoped you'd solve this one for me 👍 

We indeed needed a prepend option, and this is quite elegant. Good work.